core_backend/app/contents/models.py

```diff
@@ -121,8 +122,8 @@ async def save_content_to_db(
         language=content_language,
         content=content_db,
         content_metadata=content.content_metadata,
-        created_datetime_utc=datetime.utcnow(),
-        updated_datetime_utc=datetime.utcnow(),
+        created_datetime_utc=datetime.now(tz.utc).replace(tzinfo=None),
```

Comment:
At some stage we'll need to translate it to local time when showing it but save it as UTC

Reply:
I am not sure I understand. Why not directly get the UTC?